installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -51,30 +51,54 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
 sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack/serverless-python.yml
 
 #-------------------------------------------
-platform_services=("cognito-authorizer" "logs" "usermanagement" "services-handler" "events" "services" "logout" "login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "email" )
+platform_services=("cognito-authorizer" "platform_logs" "platform_usermanagement" "platform-services-handler" "platform_events" "platform_services" "platform_logout" "platform_login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "platform_email" )
 servicename="_services_prod"
 tablename=$stackprefix$servicename
 for element in "${platform_services[@]}"
 do
   uuid=`uuidgen -t`
-  aws dynamodb put-item --table-name $tablename --item '{
-          "SERVICE_ID":{"S":"'$uuid'"},
-          "SERVICE_CREATED_BY":{"S":"'$jazz_admin'"},
-          "SERVICE_DOMAIN":{"S":"platform"},
-          "SERVICE_NAME":{"S":"'$element'"},
-          "SERVICE_RUNTIME":{"S":"nodejs"},
-          "SERVICE_STATUS":{"S":"active"},
-          "SERVICE_METADATA":{"M":{
-                          "securityGroupIds":{"S":"'$securityGroupIds'"},
-                          "subnetIds":{"S":"'$subnetIds'"},
-                          "iamRoleARN":{"S":"'$iamRoleARN'"},
-                          "providerMemorySize":{"S":"256"},
-                          "providerRuntime":{"S":"nodejs4.3"},
-                          "providerTimeout":{"S":"160"},
-                          "runtime":{"S":"nodejs"},
-                          "service":{"S":"'$element'"},
-                          "type":{"S":"api"}
-                          }
-                  }
-          }'
+  echo -n > ./jazz-core/$element/deployment-env.yml
+  echo "service_id: "$uuid >> ./jazz-core/$element/deployment-env.yml
+ 
+  if [ element != "platform_email" ] ; then	  
+	  aws dynamodb put-item --table-name $tablename --item '{
+			  "SERVICE_ID":{"S":"'$uuid'"},
+			  "SERVICE_CREATED_BY":{"S":"'$jazz_admin'"},
+			  "SERVICE_DOMAIN":{"S":"platform"},
+			  "SERVICE_NAME":{"S":"'$element'"},
+			  "SERVICE_RUNTIME":{"S":"nodejs"},
+			  "SERVICE_STATUS":{"S":"active"},
+			  "SERVICE_METADATA":{"M":{
+							  "securityGroupIds":{"S":"'$securityGroupIds'"},
+							  "subnetIds":{"S":"'$subnetIds'"},
+							  "iamRoleARN":{"S":"'$iamRoleARN'"},
+							  "providerMemorySize":{"S":"256"},
+							  "providerRuntime":{"S":"nodejs4.3"},
+							  "providerTimeout":{"S":"160"},
+							  "runtime":{"S":"nodejs"},
+							  "type":{"S":"api"}
```

Comment:
cloud-logs-streamer is a lambda function

Reply:
In cooperated the review comment